Blockzilla/BrowserViewController.swift

```diff
@@ -206,7 +206,20 @@ class BrowserViewController: UIViewController {
         Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.click, object: TelemetryEventObject.settingsButton)
     }
 
-    fileprivate func submit(url: URL) {
+    func ensureBrowsingMode() {
+        guard !urlBar.inBrowsingMode else { return }
+
+        urlBar.isEditing = true
```

Comment:
Maybe we can make a urlBar method so that we don't have to expose the private variables, etc.? and then just call `urlBar.ensureBrowsingMode()` or whatever you want to call it